news_items/035_desktop_workspace/index.html

```diff
@@ -0,0 +1,76 @@
+<style>
+ img {
+     display: block;
+     margin-left: auto;
+     margin-right: auto;
+     margin-top: 15px;
+     margin-bottom: 15px;
+ }
+ .code-block {
+     width: 60%;
+     margin-left: auto;
+     margin-right: auto;
+     overflow: hidden;
+     margin-bottom: 1rem;
+     font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+     background: #f0f0f0;
+     padding: 1em;
+     padding-bottom: 0em;
+ }
+ .code-block div {
+     padding: 0.5rem;
+     display: block;
+     margin-bottom: -1em;
+     line-height: 125%;
+ }
+</style>
+
+<p>
+    Thanks to <a href="https://github.com/nicknytko">Nicolas Nytko</a> (CS MS '22), PrairieLearn has a new <i>desktop workspace</i> image.  This is a full Linux desktop that will allow students to run whatever graphical or terminal applications that they are provided with.  Like the existing workspaces, this desktop environment is fully persistent and is coupled with the external autograder system.
+</p>
+
+<p>
+    This new image is provided as <code class="user-output">prairielearn/workspace-desktop</code> and comes with a small set of pre-installed applications.  Installing extra software can be easily done by creating your own Docker image that uses this as a base.
+</p>
+
+<img class="border rounded shadow modal-image" src="workspace.png" width="550px">
```

Comment:
If you need to regenerate this screenshot with a new background, I'm wondering whether it would be more impactful to include the full browser window, including the PL workspace header and also the browser controls themselves. Just to drive home the point that this is _inside the browser_.